includes/polling/functions.inc.php

```diff
@@ -50,6 +53,22 @@ function poll_sensor($device, $class, $unit) {
                     }
                 }//end if
             }
+            else if ($class == "voltage") {
+                if ($device['os'] == 'hytera') {
+                    require_once "includes/polling/voltages/hytera.inc.php";
+                }
+                else {
+                    for ($i = 0;$i < 5;$i++) {
```

Comment:
Where has the code this } else { come from? It voltage classes would have just fallen back to line 94.

Probably better to do:

`} elseif ($class == "voltage" && $device['os'] == 'hytera') {`

And just run the voltage include and then leave all other os' untouched.